core/docker-compose.request-response.yaml

```diff
@@ -0,0 +1,100 @@
+version: '3.7'
+
+services:
+  redis:
+    # redis:7.0.5-alpine3.17
+    image: redis@sha256:06e4453169faaf4af93fe38689c8f8f85440990f9b9c1e91b0f9f98b1679452a
+    container_name: redis_dev
+    hostname: redis_dev
+    expose:
+      - 6379
+    healthcheck:
+      test: ["CMD", "redis-cli", "ping"]
+      interval: 5s
+      timeout: 30s
+      retries: 50
+    restart: always
+
+  listener-request-response:
+    image: orakl:v.0.0.1
+    container_name: listener_request_response_dev
+    hostname: listener_request-response_dev
+    entrypoint: ["yarn", "start:listener:request_response"]
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    environment:
+      - NODE_ENV=production
+      - ORAKL_DIR=/app/db
+      - CHAIN=baobab
```

Comment:
We should be careful with this `CHAIN` settings, later we will deploy to other chains.